src/coreclr/jit/lowerxarch.cpp

```diff
@@ -3275,6 +3275,30 @@ void Lowering::LowerHWIntrinsicGetElement(GenTreeHWIntrinsic* node)
     GenTree* op1 = node->Op(1);
     GenTree* op2 = node->Op(2);
 
+    if (op2->IsIntegralConst(0))
+    {
```

Comment:
we don't need similar change for arm64?

Reply:
No. Arm64 doesn't have as many considerations since it can't really do containment and so its already doing all the right things as far as I saw